packages/intrepid2/unit-test/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+TRIBITS_INCLUDE_DIRECTORIES(${CMAKE_SOURCE_DIR})
```

Comment:
used to allow
```c++
#include "packages/intrepid2/unit-test/Discretization/Basis/Macros.hpp"
```
in the test files, but I'm not sure it is "standard" in Trilinos to do that.